chart/dockerfiles/pgbouncer/Dockerfile

```diff
@@ -16,7 +16,7 @@
 # under the License.
 ARG ALPINE_VERSION="3.12"
 FROM alpine:${ALPINE_VERSION} AS builder
-SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
+SHELL ["/bin/ash", "-e", "-x", "-c", "-o", "pipefail"]
```

Comment:
Ash?

Reply:
Yep. https://en.wikipedia.org/wiki/Almquist_shell provided by busybox (in alpine image). Alpine has no bash. But hadolint suggest to use `ash` instead.